src/datasources/jwt/jwt.module.ts

```diff
@@ -17,7 +17,7 @@ function jwtClientFactory() {
       },
     >(
       payload: T,
-      options: { secretOrPrivateKey: string },
+      options: { secretOrPrivateKey: string; algorithm?: jwt.Algorithm },
```

Comment:
You're importing the type.

```suggestion
      options: { secretOrPrivateKey: string; algorithm?:Algorithm },
```

Reply:
Removed the import of the algorithm type, as it’s already exported by JWT.
https://github.com/safe-global/safe-client-gateway/pull/2104/commits/fc9cb89a0dce9ab682384c64089723186bee3a5a